### PR TITLE
DTFS2-4530 - Submit to UKEF confirmation page update

### PR DESCRIPTION
**Pull request by abhi-markan**
**DTFS-4530**

1. Confirmation page title and heading now show correct application type in sentence case.
2. Confirmation page text is displayed based on application submission type whether it is notice or an application.
3. Introduction of a helper function to identify application submission type.

### CODE REVIEW

File: gef-ui/server/controllers/submit-to-ukef/index.js [hunk 1]
```diff
@@ -64,7 +64,10 @@ const createSubmissionToUkef = async (req, res) => {
     await api.updateApplication(applicationId, application);
     await api.setApplicationStatus(applicationId, PROGRESS.SUBMITTED_TO_UKEF);
     // TODO: DTFS2-4706 - add a route and redirect instead of rendering?
-    return res.render('partials/submit-to-ukef-confirmation.njk', { applicationType: application.submissionType });
+    return res.render('partials/submit-to-ukef-confirmation.njk', {
+      applicationType: application.submissionType,
```
Comment (ttbarnes) [inline gef-ui/server/controllers/submit-to-ukef/index.js:68]: can we rename to `submissionType` while we're here?

File: gef-ui/server/utils/helpers.js [hunk 2]
```diff
@@ -227,6 +227,8 @@ const status = ({
 
 const stringToBoolean = (str) => (str === 'false' ? false : !!str);
 
+const isNotice = (type) => type.toLowerCase().includes('notice');
```
Comment (ttbarnes) [inline gef-ui/server/utils/helpers.js:230]: unit test

File: gef-ui/server/controllers/submit-to-ukef/index.js [hunk 3]
```diff
@@ -64,7 +64,10 @@ const createSubmissionToUkef = async (req, res) => {
...
+      isNotice: isNotice(application.submissionType),
```
Comment (ttbarnes) [inline gef-ui/server/controllers/submit-to-ukef/index.js:69]: need to update the unit test

Comment (abhi-markan) [reply gef-ui/server/controllers/submit-to-ukef/index.js:69]: Apologies mate, should have checked. Have updated.

**Review by ttbarnes — APPROVED** [review]
LGTM

**Pull request merged**